src/components/Login.vue

```diff
@@ -118,6 +118,8 @@
 <script lang="ts">
 import Vue from 'vue';
 import firebase, { User } from 'firebase/app';
+// @ts-ignore
+import introJs from 'intro.js';
```

Comment:
intro.js has typescript type definitions: https://www.npmjs.com/package/@types/intro.js
Install it and remove the @ts-ignore